src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -15,4 +27,18 @@
     <PackageReference Include="System.Memory" Version="4.5.2" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" PrivateAssets="All"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\scala\microsoft-spark-*\target\microsoft-spark-*.jar" Link="jars\%(Filename)%(Extension)">
+      <Pack>true</Pack>
+      <PackagePath>jars\%(Filename)%(Extension)</PackagePath>
+    </Content>
+    <Content Include="..\build\**" Link="build\%(Filename)%(Extension)">
```

Comment:
I think we should also include the License and Third-Parties files that we define on the root of the repo. This is a convention we've followed in other repos. https://github.com/dotnet/iot/blob/master/Directory.Build.props#L21

Reply:
I've added that as well as moved the Licence expression and some other generic bits to the props file.
Please note this means the third party notices file will show up in the Solution Explorer for each project

Reply:
How about `Visible="false"`?
```
<Content Include="$(MSBuildThisFileDirectory)..\..\THIRD-PARTY-NOTICES.TXT" Pack="true" PackagePath="\"  Visible="false"/>
```

Reply:
Yes, let's fix this on a follow up PR 😄